website/src/components/About/About.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const About = () => {
+  return (
+    <div style={{ paddingTop: 24 }}>
```

Comment:
can you add more padding on the top here? something like paddingTop: 50px I think would work better